### ISSUE TITLE
Migrate the NgRx ESLint Plugin into ngrx/platform 

### ISSUE THREAD

**Issue by timdeschryver** [ngrx/platform#3372]
### Information

Todo:

- [ ] Move https://github.com/timdeschryver/eslint-plugin-ngrx to modules/eslint-plugin
- [ ] Update docs at https://ngrx.io/guide/eslint-plugin
- [ ] Add a migration to rename rules `ngrx` becomes `@ngrx`
- [ ] Deprecate https://github.com/timdeschryver/eslint-plugin-ngrx
- [ ] Update (and decide when to run) scripts to update rules README and configs

### I would be willing to submit a PR to fix this issue

- [X] Yes
- [ ] No

**Comment by timdeschryver** [comment]
@Armenvardanyan95 do you want to do the docs part?

**Comment by Armenvardanyan95** [comment]
@timdeschryver sure, will be happy to take on the docs part!

Where can I read what exactly needs to be done?

**Comment by timdeschryver** [comment]
@Armenvardanyan95 I'll provide some more info later.
The idea is to migrate the plugin into this repository, so the guide on how to install the ESLint plugin needs to be changed. I think that you created these docs, so that's why I pinged you here, don't feel obligated to do pick this up though.

**Comment by Armenvardanyan95** [comment]
No, I haven't done anything open source for a while, I really want to pick up this one. Feel free to ping me with more details any time, I'll be happy to help 😊

**Comment by timdeschryver** [comment]
@Armenvardanyan95 this should be good to go now to add the installation page.
The installation remains the same, but the package name is now `@ngrx/eslint-plugin`.
If you don't have the time/energy to add the page that's fine to, then I'll add it later this week.

**Comment by Armenvardanyan95** [comment]
@timdeschryver I will do this today and send a PR

**Comment by Armenvardanyan95** [comment]
@timdeschryver  I am creating an installation page similar to other install guides, does the `ng add` command has specific schematics? Like adding the rules? If so, in which thread can I read about those additions so I can include them in the install guide?

**Comment by timdeschryver** [comment]
@Armenvardanyan95 correct. It also has an `ng-add` command, and it will ask you what config to use.
You can find the config [here(https://github.com/ngrx/platform/blob/master/modules/eslint-plugin/schematics/ng-add/schema.json).

**Comment by Armenvardanyan95** [comment]
Thanks, I've created a PR and added the table of configs, but I think some descriptions may need rewording https://github.com/ngrx/platform/pull/3424
